src/data/membershipCosts.js

```diff
@@ -1,4 +1,4 @@
-export const exchangeRate = 1;
+export const exchangeRate = 77;
```

Comment:
As we discussed, please do document where this magic number comes from.
Would suggest a doc comment `/** */`.